### DISCUSSION TITLE
Looking for R36.3.0 branch

### DISCUSSION THREAD

**Discussion by noob502** [OE4T/meta-tegra#1774] [General]
I looked in the 'scarthgap' branch for R36.3.0, but it seems to have updated to R36.4.0. I have tried using commit for R3.3.0 
 with SHA 8922a7e89fe962f9dabc8635610a2ca74614dab7. But the layer has a different SHA. 
 
```
 git clone https://github.com/OE4T/meta-tegra.git
cd meta-tegra
git checkout 8922a7e89fe962f9dabc8635610a2ca74614dab7
```

But the layer has a different SHA. For example, in BSP recipes, tegra-binaries-36.3.0.inc has
SRC_URI[sha256sum] = "b4656540831e74b911e2506d2c567cbb146fddd5942b675f80c2f610d6a40f43"

Is there any branch only specific to R36.3.0? if not then did I checkout right commit?

**Comment by dwalkes** [comment]
> I have tried using commit for R3.3.0
with SHA https://github.com/OE4T/meta-tegra/commit/8922a7e89fe962f9dabc8635610a2ca74614dab7

Based on https://github.com/OE4T/meta-tegra/pull/1738 I think it would be the parent of https://github.com/OE4T/meta-tegra/commit/16b0252cd7605cf35f275d3dafdc3d855ff2322f , or https://github.com/OE4T/meta-tegra/commit/8998d644ccb03ef6788258062a75154506e1ab95

> But the layer has a different SHA. For example, in BSP recipes, tegra-binaries-36.3.0.inc has
SRC_URI[sha256sum] = "b4656540831e74b911e2506d2c567cbb146fddd5942b675f80c2f610d6a40f43"

That's not related to the sha for the layer, that's the sha for the sources in the tegra-binaries package from NVIDIA used for that specific package recipe.  See https://docs.yoctoproject.org/bitbake/2.10/bitbake-user-manual/bitbake-user-manual-ref-variables.html#term-SRC_URI

> Is there any branch only specific to R36.3.0?

No, see https://github.com/OE4T/meta-tegra/wiki/Which-branch-should-I-use%3F for latest branches.  Once releases are replaced/no longer supported they generally aren't still used on branches in meta-tegra either, unless they are the end of a specific jetpack release series. In general you should avoid using old releases as you can end up in a scenario where your board doesn't boot when a new SOM is purchased.  Check PCNs from NVIDIA to make sure you support the latest hardware for a given SOM before you continue to use an old release.

**Comment by noob502** [comment]
Is there way to work with R36.3.0 ? My camera driver's is not updated with R36.4.0 yet.

**Reply by dwalkes** [reply]
See my comment above for the hash I believe you are looking for, https://github.com/OE4T/meta-tegra/commit/8998d644ccb03ef6788258062a75154506e1ab95

**Comment by noob502** [comment]
Tried both https://github.com/OE4T/meta-tegra/commit/16b0252cd7605cf35f275d3dafdc3d855ff2322f , and https://github.com/OE4T/meta-tegra/commit/8998d644ccb03ef6788258062a75154506e1ab95 but got the same error 

ERROR: /home/user01/scarthgap_36_3_0/build/../meta-tegra/recipes-kernel/linux/linux-jammy-nvidia-tegra_5.15.bb: Unable to get checksum for linux-jammy-nvidia-tegra SRC_URI entry file: file could not be found

Though I checked the files are in there

user01@u20:~/scarthgap_36_3_0/meta-tegra/recipes-kernel/linux/linux-jammy-nvidia-tegra-5.15$ tree
.
├── disable-fw-user-helper.cfg
├── disable-module-signing.cfg
├── localversion_auto.cfg
└── systemd.cfg

0 directories, 4 files
